torch/_inductor/codegen/cuda/cuda_env.py

```diff
@@ -30,3 +30,14 @@ def get_cuda_version() -> Optional[str]:
     except Exception as e:
         log.error("Error getting cuda version: %s", e)
         return None
+
+
+def nvcc_exist(nvcc_path: str = "nvcc") -> bool:
```

Comment:
Nit: `Optional[str]`. mypy will be unhappy otherwise.

Reply:
Seems that it's okay.